libkbfs/tlf_journal.go

```diff
@@ -159,6 +161,10 @@ type tlfJournal struct {
 	// Tracks background work.
 	wg kbfssync.RepeatedWaitGroup
 
+	// Non-nil when a retry has been scheduled for the future.  Should
+	// only be accessed from the background work loop goroutine.
+	retryTimer *time.Timer
```

Comment:
i think you can make this a local variable in `doBackgroundWorkLoop`, and just have a defer to stop it when the function returns (during shutdown).